### DISCUSSION TITLE
Add LDAP users button disabled

### DISCUSSION THREAD

**Discussion by mariosm1** [percussion/percussioncms#791] [Troubleshooting]
**Describe the bug**
After upgrading to 8.1.2, existing users are able to log in using ldap credentials, but the button to add new LDAP users under Administration -> Users is disabled.

**To Reproduce**
1. Set up an 8.1.2 instance
2. Configure LDAP in the config.xml
3. Try to add a new LDAP user from your chosen directory

**Expected behavior**
The Add LDAP users button should be enabled when LDAP is configured

**Comment by natechadwick** [comment]
@matomario8 Was able to reproduce this on 8.1.3 dev branch, Server Admin is unaffected but the CMS UI user import feature is not enabling like it should.

**Comment by natechadwick** [comment]
The problem appears to be in the the config.xml the DirectorySet name must start with DirectorySet:

``
<PSXDirectorySet name="DirectorySetLDAP">
`` 

You can have more than one entry they just all need to start with the DirectorySet keyword in order to be picked up as LDAP catalogers.
